addon/styles/addon.scss

```diff
@@ -35,7 +35,7 @@
         "partials/uni-header-mobile-menu",
         "partials/uni-banner",
         "partials/uni-tooltip",
-        "partials/uni-datepicker-input",
```

Comment:
nop, I just moved it to uni-input since the uni-datepicker-input file only add a "uni-input" class.